github/pulls.go

```diff
@@ -197,6 +197,7 @@ type pullRequestUpdate struct {
 }
 
 // Edit a pull request.
+// pull may be nil, in which case the requested PR is returned unmodified.
```

Comment:
Sorry I let this one slip by... Yes, I think you are right.
There really is no need to call `Edit` with `pull == nil` since `Get` could be called instead.
I'll make this an error too.